Homework/portfolio.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Experience</title>
+		<link rel="stylesheet" href="css/styles.css">
+		<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css"/>
+		<link rel="stylesheet" href="http://colorzilla.com/gradient-editor/#b4ddb4+0,83c`783+17,52b152+33,008a00+67,005700">
+
+	</head>
+	
+	<body>
+
+		<nav>
+			<a href="index.html">About Me |</a>
+			<a href="portfolio.html">Portfolio</a>
+			
+		</nav>
+
+		<h1 class="text-color">Excuse me officer...</h1>
+		
+		<p class="text-color">
+			Currenty I have been visiting public schools in the tristate area. 
+		</p>
+
+		<p class="text-color">Recently visited:
+ 			<ul class="text-color">
+ 				<li>FEWD 80</li>
+ 				<li>General Assembly seminar on life choices</li>
+ 				<li>Columbia University</li>
+ 			</ul> 
+			
+		</p>
+		<div class="center">
+			<p>
```

Comment:
@JsonLitz Here you have a `<ul>` nested inside a `<p>`. It's typical to only have `<p>`'s contain text and _inline_ elements, such as `<span>`'s or `<a>`'s. Since the `<ul>` is a block level elements, I would have probably done something like this:

```html
<p class="text-color>Recently visited:</p>
<ul>
  <li>FEWD 80</li>
  <li>etc..</li>
<ul>
```

Then, I would style the `<p>` and `<ul>` accordingly to make the look the way I wanted. I'm not saying what you did can't be done, but I rarely see `<p>`'s contain anything but text and inline elements. This might just be something you pick up having looked at a lot of code or just through trial & error in making things. Just a suggestion.